src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -396,15 +400,15 @@ csm::EcefCoord UsgsAstroFrameSensorModel::getSensorPosition(const csm::ImageCoor
 
 csm::EcefCoord UsgsAstroFrameSensorModel::getSensorPosition(double time) const {
     MESSAGE_LOG("Accessing sensor position for time {}", time);
-    if (time == m_ephemerisTime){
+    if (time == 0.0){
         csm::EcefCoord sensorPosition;
         sensorPosition.x = m_currentParameterValue[0];
         sensorPosition.y = m_currentParameterValue[1];
         sensorPosition.z = m_currentParameterValue[2];
 
         return sensorPosition;
     } else {
-        std::string aMessage = "Valid image time is %d", m_ephemerisTime;
+        std::string aMessage = "Valid image time is 0.0";
```

Comment:
Is this specific to framers since there should only ever be one time?

Reply:
Yes this is specific to the framer